### ISSUE TITLE
Different order of meta-data edit menus in page and folder

### ISSUE THREAD

**Issue by jackahl** [plone/volto#843]
The Different types of metadata you can edit are in a different order on a folder and a page.
Page:
![Screenshot edit page meta data](https://user-images.githubusercontent.com/44289551/63763925-fcfc0c80-c8c5-11e9-9222-0b038bfc1557.png)
Folder:
![Screenshot edit meta data folder](https://user-images.githubusercontent.com/44289551/63763960-0f764600-c8c6-11e9-91a7-0677edc00971.png)

**Comment by avimishra18** [comment]
Hi @jackahl, I can contribute to this issue. What should be the correct order?

**Comment by jackahl** [comment]
hey @avimishra18 I think this issue is no longer relevant as this editing view is no longer used for the page type.

**Comment by avimishra18** [comment]
Yeah, the UI has been updated. It has moved to the right-hand side, but the order remains the same. I don't know whether it still counts.

![Screenshot 2022-03-14 at 17 50 15](https://user-images.githubusercontent.com/48449499/158171006-c17c9c51-c0a4-4a82-b9bb-23a3211fce9c.png)

**Comment by jackahl** [comment]
Leave it as is, I guess. During the last 3 years I never heard someone complaining about that. So it's probably quite ok.

**Comment by sneridagh** [comment]
@avimishra18 it's a Python (backend) serializer thing in case you want to investigate.

**Comment by sneridagh** [comment]
FRT: The order of the declared behaviors in the content type affect the order of serialization.

As a thing to do, we should standardize and unify the order of the default content types.

**Comment by sneridagh** [comment]
@davisagli @mauritsvanrees I think this might be still relevant, and quite easy to solve. Something to include for 6.2?

**Comment by mauritsvanrees** [comment]
@sneridagh So you want to have the same order of behaviours in for example https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/profiles/default/types/Folder.xml and https://github.com/plone/plone.volto/blob/main/src/plone/volto/profiles/default/types/Document.xml, plus the other types?  That sounds fine.  Could also be in 6.1.  But probably only for new sites, and no upgrade step.

**Comment by sneridagh** [comment]
@mauritsvanrees Indeed, that would suffice, I guess.

FYI yesterday I started doing a bit of cleanup in Volto's issues, starting from the end :) Let's see where I can reach.

I am raising hand for those that seem reasonable and relevants.

**Comment by davisagli** [comment]
I tried to make this more consistent in https://github.com/plone/plone.volto/pull/191 and https://github.com/plone/plone.volto/pull/196 (the order of behaviors is fully specified when installing plone.volto as a distribution, but not when installing it as an add-on)
